src/main/java/com/theironyard/MessageController.java

```diff
@@ -0,0 +1,58 @@
+package com.theironyard;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Created by Zach on 10/5/16.
+ */
+@Controller
+public class MessageController {
+    @Autowired
+    MessageRepository messages;
+
+    @RequestMapping(path = "/", method = RequestMethod.GET)
+    public String home(Model model) {
+        ArrayList<Message> messageList = (ArrayList) messages.findAll();
+        Collections.sort(messageList);
```

Comment:
You know this sort can be done instead within the database easily with the right Spring Data query method name.